### DISCUSSION TITLE
Are users requesting new languages informed when I add them?

### DISCUSSION THREAD

**Discussion by keunes** [WeblateOrg/weblate#7106] [Q&A] [answered]
Hello,

I had a couple of people request new languages to be added to the AntennaPod project. I just added them (well I tried, one of three succeeded). Are these users informed that the language they requested have been added? If not, what would be the best way to let them know?

Thanks!

**Comment by nijel** [comment, marked as answer]
There is currently no specific notification on the new language request. Depending on their notification setting, users might get notified via other means (for example, as new strings to translate).
